circle.yml

```diff
@@ -49,6 +49,10 @@ test:
     - "[ -f ${CIRCLE_ARTIFACTS}/testrace.log ] && docker run -i cockroachdb/cockroach-dev shell /go/bin/go2xunit < ${CIRCLE_ARTIFACTS}/testrace.log > ${CIRCLE_TEST_REPORTS}/race/testrace.xml"
     - |
       find "${CIRCLE_ARTIFACTS}" -name '*.log' -type f -exec \
+        grep -F ': Test' | sed -E 's/(--- PASS: |\(|\))//g' | awk '{ print $2, $1 }' | sort -rn | head -n 10 \
+        > "${CIRCLE_ARTIFACTS}/slow.txt"
```

Comment:
that's only one output overwritten for multiple inputs, or am I missing something?